packages/unmock-core/src/index.ts

```diff
@@ -1,24 +1,21 @@
 // Sinon for asserts and matchers
-import * as fetch from "node-fetch";
 import * as sinon from "sinon";
-import Backend from "./backend";
-import NodeBackend from "./backend/node";
+import Backend, { buildRequestHandler } from "./backend";
 import { ILogger, IUnmockOptions, IUnmockPackage } from "./interfaces";
-import FsSnapshotter, { ISnapshot } from "./loggers/snapshotter";
-import WinstonLogger from "./loggers/winston-logger";
 import { ExtendedJSONSchema, nockify, vanillaJSONSchemify } from "./nock";
 import internalRunner, { IRunnerOptions } from "./runner";
 import { AllowedHosts, BooleanSetting } from "./settings";
 
-export { fetch };
+export * from "./interfaces";
 export * from "./types";
+export { ISnapshot } from "./interfaces";
 export { sinon };
 export { u } from "./nock";
 export { transform, Addl, Arr } from "./generator-utils";
-
-export { ISnapshot };
-const utils = { snapshotter: FsSnapshotter };
-export { utils };
+export { isRESTMethod } from "./interfaces";
```

Comment:
This and `ISnapshot` are exported with the `export * from ./interfaces`

Reply:
👍